static/bidder-info/aceex.yaml

```diff
@@ -0,0 +1,14 @@
+maintainer:
+  email: "tech@aceex.io"
+gvlVendorID: 0
```

Comment:
If you are not registered with IAB Europe for GDPR, please remove this line entirely instead of setting it to 0.

Reply:
@SyntaxNode Thank you for your review! Fixed